src/game/ZoneManager.cxx

```diff
@@ -64,7 +50,7 @@ void ZoneManager::spawn()
     std::vector<std::string> availableZoneTiles;
     for (auto &zone : node->getTileData(Layer::ZONE)->zones)
     {
-      auto &zoneTiles = TileManager::instance().getTileIDsOfCategory(zone);
+      const auto &zoneTiles = TileManager::instance().getTileIDsOfCategory(zone);
```

Comment:
Even remove reference after auto, because function already returns pointer.

Reply:
I don't get it? It does not
`std::vector<std::string> TileManager::getTileIDsOfCategory(Zones zone)`

Reply:
Sorry, my bad.